backend/src/domain/models/Game.js

```diff
@@ -35,12 +35,11 @@ export class Game {
   }
 
   /**
-   * This disconnects the client side user from the lobby
+   * This emits an end_game event to all users in the session
    */
   endGame() {
-    this.session.users.forEach((e) => {
-      e.socket.disconnect(true);
+    this.session.users.forEach((user) => {
+      user.socket.emit('end_game');
```

Comment:
I think you can use io.to(roomid or gameid).emit to emit to all sockets in the room. Instead of looping through

Reply:
okay, so do you think i should still change for conventions sake

Reply:
Yea